src/Landing/index.jsx

```diff
@@ -0,0 +1,72 @@
+import React, { Component } from "react";
+import FacebookButton from "../auth/socialAuth/facebook/components/Facebook";
+import GoogleButton from "../auth/socialAuth/google/components/Google";
+import axios from "axios";
+
+let errorMessage = "";
```

Comment:
Put this in the component's state instead

Reply:
Kindly advise me on how to do this please, I have tried and have failed.

Reply:
@AGMETEOR I have added a state please do check it out.